src/redux/actions.js

```diff
@@ -19,8 +18,3 @@ export const listenToQueries = () => (
     });
   }
 );
-
-export const deleteVisible = (index) => ({
```

Comment:
They do but i max them at 40, after that they get trimmed. I can do it more elegantly but the perf doesnt really suffer I think. I will probably let them expire after their animation again after i clean up Pod.js etc.